src/client/screens/ImportProjectScreen.tsx

```diff
@@ -33,39 +50,50 @@ const validate = (
   const regionConfig = importResource.data;
   const districtsDefinition = "resource" in importResource ? importResource.resource : null;
   const numberOfDistricts = form.numberOfDistricts;
+  const populationDeviation = form.populationDeviation || DEFAULT_POPULATION_DEVIATION;
   return numberOfDistricts &&
     maxDistrictId &&
     regionConfig &&
+    populationDeviation &&
     districtsDefinition &&
     numberOfDistricts >= maxDistrictId
-    ? { numberOfDistricts, regionConfig, districtsDefinition, valid: true }
-    : { numberOfDistricts, regionConfig, districtsDefinition, valid: false };
+    ? { numberOfDistricts, regionConfig, districtsDefinition, populationDeviation, valid: true }
+    : { numberOfDistricts, regionConfig, districtsDefinition, populationDeviation, valid: false };
 };
 
 type ImportResource = WriteResource<IRegionConfig | null, DistrictsDefinition>;
 
 interface ConfigurableForm {
   readonly numberOfDistricts: number | null;
+  readonly isCustom?: boolean;
+  readonly populationDeviation: number;
 }
 
 type ProjectForm = ValidForm | InvalidForm;
 
 interface ValidForm {
   readonly regionConfig: IRegionConfig;
+  readonly chamber?: IChamber;
   readonly districtsDefinition: DistrictsDefinition;
   readonly numberOfDistricts: number;
+  readonly isCustom?: boolean;
+  readonly populationDeviation: number;
   readonly valid: true;
 }
 
 interface InvalidForm {
   readonly regionConfig: IRegionConfig | null;
+  readonly chamber?: IChamber;
   readonly districtsDefinition: DistrictsDefinition | null;
   readonly numberOfDistricts: number | null;
+  readonly isCustom?: boolean;
+  readonly populationDeviation: number;
```

Comment:
You should match the existing style of the surrounding code - it uses `null` for missing data, not `undefined` (which is what using `?` implies).
Additionally
 - it is possible for someone to leave the population deviation field empty, so we should represent that state in our `InvalidForm` type.
 - `isCustom` shouldn't be optional
```suggestion
  readonly chamber: IChamber | null;
  readonly districtsDefinition: DistrictsDefinition | null;
  readonly numberOfDistricts: number | null;
  readonly isCustom: boolean;
  readonly populationDeviation: number | null;
```